migrations/versions/2023_06_11_09_09_00_8337558f4f36_move_deleted_utc_into_state_user_.py

```diff
@@ -0,0 +1,46 @@
+"""Move 'deleted_utc' into 'state_user_deleted_utc'.
+
+Revision ID: 8337558f4f36
+Revises: 6403c9151c12
+Create Date: 2023-06-11 09:09:00.235602+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '8337558f4f36'
+down_revision = '6403c9151c12'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('comments', sa.Column('state_user_deleted_utc', sa.DateTime(), nullable=True))
```

Comment:
Postgres has both `timestamp [without time zone]` and `timestamp with time zone`. These are the same size on disk. SQLA turns the former into a tz-naive datetime and the latter into a tz-aware datetime. The latter are often much easier to work with, in my experience, since Python's stdlib `datetime` treats them both differently (despite both being `datetime.datetime` objects) and sometimes requires tz-awareness (which you will only discover at runtime).

This migration appears to create `timestamp without time zone` columns. Would recommend finding a way to make them tz-aware.

Reply:
Sure, done!